src/Elastic.Apm/Agent.cs

```diff
@@ -62,6 +63,17 @@ public static class Agent
 		/// </summary>
 		public static ITracer Tracer => Instance.Tracer;
 
+		/// <summary>
+		/// Sets up multiple <see cref="IDiagnosticsSubscriber" />'s to start listening to one or more <see cref="IDiagnosticListener" />'s
+		/// </summary>
+		/// <param name="subscribers">
+		/// An array of <see cref="IDiagnosticSubscriber" /> that will set up <see cref="IDiagnosticListener" /> subscriptions
+		/// </param>
+		/// <returns>
+		/// A disposable referencing all the subscriptions, disposing this is not necessary for clean up only to unsubscribe if desired.
+		/// </returns>
+		public static IDisposable Subscribe(params IDiagnosticsSubscriber[] subscribers) => Instance.Subscribe(subscribers);
+
 		public static void Setup(AgentComponents agentComponents)
```

Comment:
Could potentially also add subscribers to `Setup()`, maybe something for a follow up PR.

Reply:
👍  But I think for now this is fine.